src/app/service/shoppingcart.service.ts

```diff
@@ -1,4 +1,4 @@
-import { Injectable } from '@angular/core';
+import {Injectable} from '@angular/core';
```

Comment:
I would like you to agree to a standard for this (`{ Injectable }` or `{Injectable} `) and be consistent throughout the project.